web/themes/custom/votegov/src/sass/uswds-overrides/usa-form.scss

```diff
@@ -22,9 +22,16 @@
       height: 40px !important;
     }
   }
+
+  a:where(:not(.usa-button)):is(:active, :focus) {
```

Comment:
@clmedders this seems overcomplicated. Can you identify an existing style that we can update to make sure that the style is applied uniformly?